cherrypy/test/test_httputil.py

```diff
@@ -30,3 +31,17 @@ def test_urljoin(self, script_name, path_info, expected_url):
         """Test all slash+atom combinations for SCRIPT_NAME and PATH_INFO."""
         actual_url = httputil.urljoin(script_name, path_info)
         assert actual_url == expected_url
+
+
+def test_valid_status():
+    code, reason, message = httputil.valid_status(200)
+    assert code == 200 and reason == 'OK'
```

Comment:
Could you please rewrite this using `@pytest.parametrize`, so that the test body would be simpler like one above